python/src/trezorlib/cli/settings.py

```diff
@@ -95,24 +95,28 @@ def cli() -> None:
 
 
 @cli.command()
-@click.option("-r", "--remove", is_flag=True)
```

Comment:
Please keep the `-r` option functional for backwards compatibility.

All of these should work:
```
trezorctl set pin on
trezorctl set pin
trezorctl set pin off
trezorctl set pin -r
```
I'd ignore the `set pin off -r` case, but raise on `set pin on -r`

dtto for wipe-code

Reply:
that would be nice but tricky

alright, how about this:

- define a global choice group where "enable/enabled/on" match True and "disable/disabled/off" match False. Use it everywhere.
- convert `passphrase_enable()` and `passphrase_disable()` to a single command where the on/off thing is a parameter
- create a new command `passphrase_new_command()` in `trezorctl.py`, that will be aliased to `enable-passphrase` AND `disable-passphrase`, and it will only tell the user to use "set passphrase on/off" and exit with an error.

Reply:
make sure to set `hidden=True` for the `passphrase_new_command`

it would be cool if the help page still only listed `[on|off]`
maybe something like:
```python
def argument_enable(default: bool = True, **kwargs) -> Callable:
	return click.argument("enable", Choice({"on": True, ...), metavar="[on|off]", default=default)

@click.command()
@argument_enable()
def wipe_code(enable, ...):
    ...
```

Reply:
alternately, to keep the backwards compatibility even more: `passphrase_new_command` could read its own name from context (`@click.pass_context`) and invoke the appropriate passphrase call (https://click.palletsprojects.com/en/8.0.x/advanced/#invoking-other-commands)

Reply:
I will see what I can do.

So far [01b9d18](https://github.com/trezor/trezor-firmware/pull/2123/commits/01b9d18232dbcbdd74310daa1ffd6f1cc98491de) should contain the unification to `on/off`